build/firmware/Common.mk

```diff
@@ -302,6 +347,26 @@ $(VERSIONS):
 	  echo EDK2-Platforms diff end >> $(CURDIR)/$@
 	popd
 
+$(TA_VERSIONS):
+	@echo Logging version information of TA repositories
+	rm -f $@
+	echo TA commit information: >> $@
+	pushd $(TA_ROOT) && \
+	  git config --get remote.origin.url >> $(CURDIR)/$@ && \
+	  git rev-parse HEAD >> $(CURDIR)/$@ && \
+	  echo TA diff information: >> $(CURDIR)/$@ && \
+	  git diff HEAD >> $(CURDIR)/$@ && \
+	  echo TA diff end >> $(CURDIR)/$@
+	popd
+
+place_authvars_notice:
+	rm -f $(AUTHVARS_BIN_PLACE)/NOTICE
+	echo "PUT AUTHVARS NOTICE HERE!" >> $(AUTHVARS_BIN_PLACE)/NOTICE
+
+place_ftpm_notice:
+	rm -f $(FTPM_BIN_PLACE)/NOTICE
+	echo "PUT FTPM NOTICE HERE!" >> $(AUTHVARS_BIN_PLACE)/NOTICE
```

Comment:
What is the notice supposed to be?

Reply:
We need to distribute a 3rd party notice with binaries. This is just a placeholder until the Notices file is present in the MSRsec repo.